### ISSUE TITLE
Use of 365 ProPlus on a personal home computer

### ISSUE THREAD

**Issue by BB502** [MicrosoftDocs/OfficeDocs-DeployOffice#800]
**TL;DR** - _Enterprise-licensed Office 365 ProPlus on a personal home computer - what (if any) privacy is the user giving up by taking up the offer of a "free" home copy of Office?_
_________________________

So, let's say a user installs Office 365 ProPlus on their personal home computer, using one of the 5 installs permitted by their work account (by downloading it themselves from portal.office.com), but only uses it for personal reasons (no work documents, no use of OneDrive for Business or Sharepoint etc).

In this scenario, what data is sent back to their employer (the admin of the work account used to do the download, that the Office 365 apps are licensed to)?

I'm assuming (and please correct me if I'm being naive!) that personal documents saved locally or to a personal OneDrive aren't sent, but what about telemetry data like file names, spellcheck usage, information about the computer itself, and other potentially personally identifying information getting either sent to the employer, or tied to the user's corporate Microsoft account and profile? 

Or is all usage completely private to the user, so the employer/admin is "blind" to the usage, with the exception of simply the licensing information saying that a licensed install exists?

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 19062a9a-86ed-a800-c7bd-07a089ec72a9
* Version Independent ID: 18ea08b3-68a3-e957-05f9-10157af5a737
* Content: [About Microsoft 365 Apps in the enterprise](https://docs.microsoft.com/en-gb/deployoffice/about-microsoft-365-apps)
* Content Source: [DeployOffice/about-microsoft-365-apps.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/about-microsoft-365-apps.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @jwhit-MSFT
* Microsoft Alias: **jwhit**

**Comment by manoth-msft** [comment]
There are multiple layers to this question: The client will e.g. send the computer's name and OS version to the Office service in order to track the activation. This data is visible to the user, but not the admin. It will store the list of most recently used files, some app settings, etc. in the enterprise OneDrive, this data is stored in the employer's tenant, but not visible to the admin. Telemetry data (like MRU, Add-ins, document names) is only sent back to the employer, if the admin has enabled Telemetry (it's off by default) through OCPS. But in most cases, a personal, non-corp machine will not be able to upload the data to the corp network anyway.

https://docs.microsoft.com/en-us/deployoffice/licensing-activation-data-sent-microsoft-365-apps
https://docs.microsoft.com/en-us/DeployOffice/compat/deploy-telemetry-dashboard

**Comment by BB502** [comment]
Thanks for the quick reply @manoth-msft - so, assuming that the non-corp machine has never been touched by the corp, and the only "link" is the installing of apps through the Office Portal, then no telemetry data would get sent even if telemetry was enabled on the OCPS side, and therefore there's no privacy concern for the user to worry about? (And can write his resume at home in the evenings without his employer finding out 😆?)

**Comment by manoth-msft** [comment]
With OCPS, the policies are bound to the user identity. So if someone on a personal device downloads, installs and activates a corp-licensed Office, these policies will apply, because the user has to sign in with corp credentials to actually activate Office (or when downloaded from portal, we will automate this). In this case, each Office app session might to subject to policies like "Enable Telemetry" and "Upload data to [local folder\UNC path]". So technically Office will e.g. record the information that "resume.docx" just crashed and tries to upload this data to the given location. As the target location for Telemetry is a local folder or a network share, it is very unlikely, that the personal device can access and log into the given path and actually upload the data for further processing by corp.

But, if a corp enabled Telemetry data collection, they should rules in place who can access the data for which pruposes.

**Comment by manoth-msft** [comment]
Does this answer your questions and can we close this item?

**Comment by BB502** [comment]
Yes, thank you for your help @manoth-msft, much appreciated 😃
